src/Avalonia.Build.Tasks/Program.cs

```diff
@@ -1,70 +1,64 @@
 using System;
-using System.Collections;
 using System.IO;
-using System.Linq;
 using Microsoft.Build.Framework;
 
 namespace Avalonia.Build.Tasks
 {
     public class Program
     {
+        private const string _originalDll = "original.dll";
+        private const string _references = "references";
+        private const string _outDll = "out.dll";
+
         static int Main(string[] args)
         {
-            if (args.Length != 3)
+            if (args.Length != 1 && args.Length != 3)
             {
-                if (args.Length == 1)
-                    args = new[] {"original.dll", "references", "out.dll"}
-                        .Select(x => Path.Combine(args[0], x)).ToArray();
-                else
-                {
-                    Console.Error.WriteLine("input references output");
-                    return 1;
-                }
+                const string referencesOutputPath = "path/to/Avalonia/samples/Sandbox/obj/Debug/netcoreapp3.1/Avalonia";
+                Console.WriteLine(@$"Usage:
+    1) dotnet ./Avalonia.Build.Tasks.dll <ReferencesOutputPath>
+       , where <ReferencesOutputPath> likes {referencesOutputPath}
+    2) dotnet ./Avalonia.Build.Tasks.dll <AssemblyFilePath> <ReferencesFilePath> <OutputPath>
+       , where:
+           - <AssemblyFilePath> likes {referencesOutputPath}/{_originalDll}
+           - <ReferencesFilePath> likes {referencesOutputPath}/{_references}
+           - <OutputPath> likes {referencesOutputPath}/{_outDll}");
+                return 1;
```

Comment:
It looks like that lines 17-26 are the only relevant changes in this PR. Do other parts of the diff have meaning other than moving code around?

Reply:
Left this

Reply:
Thanks